wormhole-connect/src/views/v2/Redeem/index.tsx

```diff
@@ -273,6 +274,13 @@ const Redeem = () => {
   // Checks whether the receiving wallet is currently connected
   const isConnectedToReceivingWallet = useMemo(() => {
     if (!recipient) {
+      // For Solana transfers, the associated token account (ATA) might not exist,
+      // preventing us from retrieving the recipient wallet address.
+      // In such cases, when resuming transfers, we allow the user to connect a wallet
+      // to claim the transfer, which will create the ATA.
+      if (isResumeTx && toChain === 'Solana' && receivingWallet.address) {
```

Comment:
so if they connect to the wrong Solana wallet, are you just relying on the transaction failing?

Reply:
Going to make this more robust and warn the user if the wrong wallet is connected (we can derive the ATA and see if it matches what's on the VAA)

Reply:
yeah I was gonna suggest that but maybe it's not necessary.

Reply:
I'd like to avoid having Connect ever look at VAAs 😬